FirebaseMessaging/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 2021-04 -- v7.11.1
```

Comment:
I think this should be under v7.11.0. Thanks!

Reply:
It's a Firebase wide change so it should go to FirebaseCore/CHANGELOG.

Also we typically don't highlight formatting changes in Changelogs.

Reply:
👍 feel free to omit this change since 7.11.0 is not empty and clang-format changes aren't noticeable by users unless they go read the source.

Reply:
And it's probably sufficient for those reading the source to check the git logs. 

In summary, thanks for the PR, but recommend closing without merging.

Reply:
got it. Thanks for the inputs here folks :)